### PR TITLE
support for fine tuning jobs

### PR DESCRIPTION
**Pull request by lancecarlson**
## All Submissions:

* [ X ] Have you followed the guidelines in our [Contributing document](../blob/main/CONTRIBUTING.md)?
* [ X ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [ X ] Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

**Comment by lancecarlson** [comment]
I tried to make this as close to the existing API's as possible. I ran into some issues running the tests all at once because you have to wait for the file to process before you can run the fine tuning job. Then on top of that you have to wait 10 minutes to run another test because you can't run more than 1 simultaneous fine tune job. Not sure how you want to fix this with CI.

File: README.md [hunk 1]
```diff
@@ -325,6 +325,52 @@ client.files.content(id: "file-123")
...
+```
+
+You can then use this file ID to create a fine tuning job:
+
+```ruby
+response = client.fine_tuning_job.create(
+    parameters: {
+    training_file: file_id,
+    model: "gpt-3.5-turbo-0613"
+})
```
Comment (alexrudall) [inline README.md:344]: Indentation

**Review by alexrudall — CHANGES_REQUESTED** [review]
This is looking really nice - thanks so much for your work on this. Couple of changes

- I prefer pluralized resources, could you change to `fine_tuned_jobs`?
- Really appreciate you taking the time to add to the README. I haven't read much on this yet, do fine-tune-jobs replace fine-tunes? If not would be good to explain in the README when to use a job and when not to.

**Comment by alexrudall** [comment]
RE: tests, I aim for

- No use of sleep
- All tests can be run at once and pass with and without VCR 

This may mean just not including some tests, which I prefer to flaky or slow tests. Or just tests that the endpoint responds with an error.

**Comment by lancecarlson** [comment]
I've been having a lot of issues with the tests because of the 1 job at a time limit. I don't think it will be possible to make the tests successfully run with and without vcr consistently. It seems a shame to completely remove the tests for this, but I'm not sure I see a good way around it. Any thoughts/ideas? I've tested that they work in the irb pretty extensively.

**Comment by alex-areshnikov** [comment]
@alexrudall @lancecarlson Thank you for your work! Any update on this MR? Would be really nice to see it merged/implemented. Thanks!

**Comment by lancecarlson** [comment]
@alex-areshnikov this works in production, it's the tests that are holding this up. Want to take a look at it? :)

**Comment by alexrudall** [comment]
Thanks for your work on this @lancecarlson - released in [v6](https://github.com/alexrudall/ruby-openai/releases/tag/v6.0.0)

**Pull request closed without merging**